eslint/babel-eslint-parser/test/index.js

```diff
@@ -1,19 +1,41 @@
-import assert from "assert";
 import espree from "espree";
```

Comment:
If we are removing this from `package.json`, we should explicitly import the correct one.

```js
const espreePath = require.resolve("espree", {
  paths: [path.dirname(require.resolve("eslint"))]
]);
const espree = require(espreePath);
```

Reply:
That makes sense!